bootstrap/node.go

```diff
@@ -43,7 +43,7 @@ func NewNode(nodeConfig config.NodeConfig, logger log.Logger) *Node {
 	ctx, ctxCancel := context.WithCancel(context.Background())
 	config.NewValidator(logger).ValidateMainNode(nodeConfig) // this will panic if config does not pass validation
 
-	nodeLogger := logger.WithTags(log.Node(nodeConfig.NodeAddress().String()))
+	nodeLogger := logger.WithTags(log.Node(nodeConfig.NodeAddress().String()), log.Uint32("virtual-chain-id", uint32(nodeConfig.VirtualChainId())))
```

Comment:
We already log virtual chain id (and there is special log field for it): https://github.com/orbs-network/orbs-network-go/blob/36e21b654a918ecc6a31b121022a9991897ef267/instrumentation/logger.go#L64

Reply:
interesting
I remembered we used to, but when I looked at the log I couldn't find vcid anywhere
perhaps vcid is defined at the `main.go` level and the in-process E2Es don't use `main.go`

@ronnno Let's use the `logfields.VirtualChainId` tag but remove its usage from main, leaving it at the `node.go` level instead.

Reply:
Please review...